keras_model.py

```diff
@@ -0,0 +1,36 @@
+from keras.models import load_model
+from multiprocessing import Lock
+from multiprocessing.managers import BaseManager
+
+import config
+cfg = config.get_localized_config()
+
+class KerasModel():
+    def __init__(self):
+        self.mutex = Lock()
+        self.model = None
+
+    def initialize(self):
+        self.model = load_model(cfg.model_path)
+        self.model._make_predict_function()
```

Comment:
Leave a comment explaining why we're doing this and linking to https://stackoverflow.com/questions/43136293/ please, since it is otherwise pretty odd to be calling a "private" method of the model like this.